pkg/cluster/internal/providers/podman/node.go

```diff
@@ -63,7 +63,10 @@ func (n *node) IP() (ipv4 string, ipv6 string, err error) {
 	if len(lines) != 1 {
 		return "", "", errors.Errorf("file should only be one line, got %d lines", len(lines))
 	}
-	ips := strings.Split(lines[0], ",")
+	// TODO: investigate where the double quotes are added
+	// it does not seem to happen running from the CLI
```

Comment:
a call specifying format has different output with TTY or not!?

Reply:
solved removing the quotes and using a rune